app/frontend/components/shared/UnsavedChangesAlert.tsx

```
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Alert, AlertDescription } from '@/components/ui/alert'
+import { CircleAlert } from 'lucide-react'
+
+interface UnsavedChangesAlertProps {
+  originalData: Object
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)